tests/unit/test_cli.py

```diff
@@ -231,7 +229,7 @@ class DecideColorizationTestcase:
 
 
 @pytest.mark.parametrize(
-    ('testcase',),
+    ('testcase',),  # noqa: PT006
```

Comment:
No idea how to solve this.
PT006: `Wrong name(s) type in `@pytest.mark.parametrize`, expected `csv``

Reply:
Hmm, no idea. A tuple of strings should be acceptable, or a coma-separated string, `"foo, bar"`.

Reply:
I wouldn't mind, although I for one never used it, all pytest versions I have experience with were fine with `"foo, bar"` or `('foo', 'bar')`. I never had to name the first argument :)

Reply:
ok, thanks. Using a string without argument names then.